portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -14,6 +14,10 @@
 
 package com.google.sps.servlets;
 
+import java.io.*; 
+import com.google.sps.data.Comment;
```

Comment:
The code is failing to compile for me. This doesn't seem to exist - did you add this file and forget to git add it, perhaps?